### ISSUE TITLE
Separate update site? Or part of Fiji core?

### ISSUE THREAD

**Issue by ctrueden** [morphonets/SNT#61]
The README says that SNT is distributed with Fiji, but that's not currently true: one must enable the Neuroanatomy update site.

The intent is to continue (for now) distributing SNT on its own update site, not part of Fiji core, correct?

See also https://gitter.im/fiji/fiji?at=613a57b2f3449e13a4645e1e

**Comment by tferr** [comment]
Correct. We did flirt with the idea of having it in core Fiji, but realized soon it would be too involved.  The only motivation to have it in core Fiji was that Simple Neurite Tracer (that we now replace) was part of Fiji, and we were afraid of breaking backwards functionality. But the way we have it now seems to work fine: before archiving Simple Neurite Tracer we made it so that users get prompted  to subscribe to the Neuroanatomy update site when trying to call the archived plugin. It has been likely this for almost a year, and we did not get any complaints on that, so I think it worked out well.

So there is really no major reason to move it to core Fiji. Folks that don't use SNT would complain of the full submenu real-estate we occupy :)

Actually, I would try to convince anyone with neuro-centric software to use the Neuroanatomy update site, so that, from the end-user point of view, all things Neuroanatomy get served under the same subscription. Such 'thematic' subscription would certainly help to clean up some menu clutter.

I updated the README.
